### PR TITLE
[ACTIONS] Project broadcast new actions

### PR DESCRIPTION
**Pull request by jcortes**
### <samp>🤖 Generated by Copilot at 8e5a06d</samp>

This pull request adds a new app module and three action components for the Project Broadcast app, which is a service for sending text messages to contacts. The app module defines the app props and methods, using the axios module and a constants module. The action components allow users to create contacts, add keywords to contacts, and broadcast messages to contacts, using the app methods. The pull request also updates the package.json and pnpm-lock.yaml files to reflect the new dependency on the @pipedream/platform module, and deletes some unused files.

### <samp>🤖 Generated by Copilot at 8e5a06d</samp>

*  Add three new action components for the Project Broadcast app: `add-keyword-to-contact`, `broadcast-message`, and `create-contact` ([link](https://github.com/PipedreamHQ/pipedream/pull/7997/files?diff=unified&w=0#diff-6073ce262c4b668d111704d3c480f5f2e5c28747d5ddb1dcc5f7c112eedfaeecR1-R50), [link](https://github.com/PipedreamHQ/pipedream/pull/7997/files?diff=unified&w=0#diff-ca708fb7f615f8cbd97c668b502c7e947297b16f827d7e3199a0b544cdde3195R1-R53), [link](https://github.com/PipedreamHQ/pipedream/pull/7997/files?diff=unified&w=0#diff-99f1bcbf1b8bc2d9e809058434a993a4801802c9531ae563291f912fe18c32c8R1-R86))
* Add a new app module for the Project Broadcast app: `project_broadcast.app.mjs` ([link](https://github.com/PipedreamHQ/pipedream/pull/7997/files?diff=unified&w=0#diff-993c717320c4db98e033fb200c5c6b82342e969d9a848542734c44173aee8b1eR1-R108))
* Add a new module for the Project Broadcast app that exports some common constants, such as the base URL and the version path of the API: `constants.mjs` ([link](https://github.com/PipedreamHQ/pipedream/pull/7997/files?diff=unified&w=0#diff-daf13710bfaec2ea96b5394f71d60f646bdebe48814261b71a0c985c07e8c80bR1-R7))
* Update the package.json file for the Project Broadcast app to reflect the new version, main file, and dependency on the @pipedream/platform module ([link](https://github.com/PipedreamHQ/pipedream/pull/7997/files?diff=unified&w=0#diff-9528d23095436b4e815702f0a10e8802a3fda1845eb41db279bdc94ad9c315e3L3-R18))
* Update the pnpm-lock.yaml file to include the dependency on the @pipedream/platform module for the Project Broadcast app ([link](https://github.com/PipedreamHQ/pipedream/pull/7997/files?diff=unified&w=0#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bbL3999-R4003))
* Delete the unused files `project_broadcast.app.ts` and `.gitignore` ([link](https://github.com/PipedreamHQ/pipedream/pull/7997/files?diff=unified&w=0#diff-00fcc28417bec7e0a4e8918343aa716f418baea599f657fd8a915c29b5c08130), [link](https://github.com/PipedreamHQ/pipedream/pull/7997/files?diff=unified&w=0#diff-8dd8733f89467a0b559fb0f7c9db0bd842e3c83a7e188cc93fca759e979bfa88))

### CODE REVIEW

File: components/project_broadcast/actions/add-keyword-to-contact/add-keyword-to-contact.mjs [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    } = {}) {
+      return this.app.put({
+        path: `/contacts/${contactId}/keywords/${keywordId}/apply`,
+        ...args,
+      });
+    },
+  },
+  run({ $: step }) {
+    const {
+      // eslint-disable-next-line no-unused-vars
```
Comment (vellames) [inline components/project_broadcast/actions/add-keyword-to-contact/add-keyword-to-contact.mjs:36]: Hi there. Some specific reason for this disable?

File: components/project_broadcast/actions/broadcast-message/broadcast-message.mjs [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    broadcastMessage(args = {}) {
+      return this.app.post({
+        path: "/broadcasts",
+        ...args,
+      });
+    },
+  },
+  run({ $: step }) {
+    const {
+      // eslint-disable-next-line no-unused-vars
```
Comment (vellames) [inline components/project_broadcast/actions/broadcast-message/broadcast-message.mjs:41]: Hi there. Some specific reason for this disable?

File: components/project_broadcast/actions/create-contact/create-contact.mjs [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    createContact(args = {}) {
+      return this.app.post({
+        path: "/contacts",
+        ...args,
+      });
+    },
+  },
+  run({ $: step }) {
+    const {
+      // eslint-disable-next-line no-unused-vars
```
Comment (vellames) [inline components/project_broadcast/actions/create-contact/create-contact.mjs:74]: Hi there. Any specific reason for this disable?

**Review by vellames — CHANGES_REQUESTED** [review]
Hello. I just left some comments

**Review by michelle0927 — CHANGES_REQUESTED** [review]
Looks good, just one comment.

File: components/project_broadcast/actions/add-keyword-to-contact/add-keyword-to-contact.mjs [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+export default {
+  key: "project_broadcast-add-keyword-to-contact",
+  name: "Add Keyword To Contact",
+  description: "Add a keyword to a contact. [See the documentation](https://www.projectbroadcast.com/apidoc/#api-Contacts-Apply_Keyword)",
+  type: "action",
+  version: "0.0.1",
+  props: {
+    app,
+    contacts: {
```
Comment (michelle0927) [inline components/project_broadcast/actions/add-keyword-to-contact/add-keyword-to-contact.mjs:11]: Looks like this should be `contactId`.
```suggestion
    contactId: {
```

**Review by michelle0927 — CHANGES_REQUESTED** [review]

**Review by michelle0927 — APPROVED** [review]
Approved

**Review by lcaresia — COMMENTED** [review]
Ignore this comment, missclick.

**Pull request merged**